sechub-cli/src/mercedes-benz.com/sechub/util/errorhandler.go

```diff
@@ -28,8 +28,12 @@ func HandleHTTPError(err error, exitCode int) {
 // HandleHTTPResponse - handler method for http response. when not 200 an error log entry will be created and sechub client does exit
 func HandleHTTPResponse(response *http.Response, exitCode int) {
 	if response.StatusCode >= 400 { // StatusCode is 4xx or 5xx
-		b, _ := ioutil.ReadAll(response.Body)
-		LogError(fmt.Sprintf("The HTTP request failed with status code '%s'\nbody=%s\n", response.Status, string(b)))
+		bodytext, _ := ioutil.ReadAll(response.Body)
+		LogError(fmt.Sprintf("The SecHub server responded with HTTP status code '%s'\nbody=%s", response.Status, string(bodytext)))
+		switch response.StatusCode {
```

Comment:
I thought about an if statement also, but then decided to create a structure that makes extensions for other status-code-based messages easy.

@winzj @haerter-tss How do you think about this?

Reply:
If there's a possibility that more status codes need to be added in the future I would prefer Sven's approach. Otherwise I'm with Jeremias - an if-statement would be more readable.

Reply:
@sven-dmlr in this case the switch statement is fine.